recipes/azure-datalake-store/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "azure-datalake-store" %}
+{% set version = "0.0.47" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 55bcbec99a35a52ea291518dadfd94c3b645e66d05ce927228289b93979ad3a5
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cffi
+    - adal >=0.4.2
+    - requests >=2.20.0
+
+test:
+  imports:
+    - azure.datalake.store
+
+about:
+  home: http://github.com/Azure/azure-data-lake-store-python
+  license: MIT
+  license_family: MIT
+  license_file: License.txt
+  summary: 'This project is the Python filesystem library for Azure Data Lake Store'
+
+  description: |
+    A pure-python interface to the Azure Data-lake Storage system,
+    providing pythonic file-system and file objects, seamless transition
+    between Windows and POSIX remote paths, high-performance up- and down-loader.
+  doc_url: http://github.com/Azure/azure-data-lake-store-python
+  dev_url: http://github.com/Azure/azure-data-lake-store-python
+
+extra:
+  recipe-maintainers:
+     - dillonroach
+     - scopatz
+     - costrouc
```

Comment:
@scopatz and @costrouc 
Are you willing to be a maintainer of this recipe as well?

Reply:
Yes

Reply:
yep